.github/workflows/build-wheels-publish.yml

```diff
@@ -11,14 +11,6 @@ on:
         options:
           - '0'
           - '1'
-      fullMode:
```

Comment:
Why does this get removed?

Reply:
Because it’s no more needed, now all python versions will be tested